_posts/en/newsletters/2024-08-09-newsletter.md

```diff
@@ -0,0 +1,391 @@
+  reconstructions he's observed using his own nodes with various
+  settings, with some data going back about six months.  Recent data on
+  the effect of enabling `mempoolfullrbf` only goes back about a week,
+  but it matches Maxwell's report.  This has helped motivate
+  consideration of a [pull request][bitcoin core #30493] to enable
+  `mempoolfullrbf` by default in an upcoming version of Bitcoin Core.
+
+<!-- FIXME:harding update ephemeral anchors topic to include P2A -->
+
+- **Replacement cycle attack against pay-to-anchor:** Peter Todd
+  [posted][todd cycle] to the Bitcoin-Dev mailing list about the
+  pay-to-anchor (P2A) output type that is part of the [ephemeral
+  anchors][topic ephemeral anchors] proposal.  P2A is an transaction
+  output that anyone can spend.  This can be useful for [CPFP][topic
+  cpfp] fee bumping---especially in multiparty protocols such as LN.
+  However, CPFP fee bumping in LN is currently vulnerable to a
+  counterparty performing a [replacement cycling attack][topic
+  replacement cycling] where the malicious counterparty perfroms a
+  two-step process.  They first replace an honest user's version of a
+  transaction with the counterparty's version of the same transaction.
+  They then replace the replacement with a transaction unrelated to
+  either user's version of the transaction.  When an LN channel has
+  unresolved [HTLCs][topic htlc], a successful replacement cycle can
+  allow the counterparty to steal from the honest party.
+
+  Using LN's current [anchor outputs][topic anchor outputs] channel
+  type, only a counterparty can perform a replacement cycling attack.
+  However, Todd points out that, because P2A allows anyone to spend the
+  output, anyone can perform a replacement cycling attack against
+  transactions using it.  Still, only a counterparty can economically
+  benefit from the attack, so there's no direct incentive for third
+  parties to attack P2A outputs.  The attack can be free in the case
+  where the attacker is planning to broadcast their own transaction at a
+  feerate higher than the honest user's P2A spend and the attacker is
+  able to successfully complete the replacement cycle without their
+  intermediate state getting confirmed by miners.  All existing deployed
+  LN mitigations against replacement cycling attacks (see [Newsletter
+  #274][news274 cycle mitigate]) will be equally effective at defeating
+  P2A replacement cycling.
+
+- **Proposed BIP for scriptless threshold signatures:** Sivaram
+  Dhakshinamoorthy [posted][dhakshinamoorthy frost] to the Bitcoin-Dev
+  mailing list to announce the availability of a [proposed BIP][frost
+  sign bip] for creating [scriptless threshold signatures][topic
+  threshold signature] for Bitcoin's implementation of [schnorr
+  signatures][topic schnorr signatures].  This allows a set of wallets
+  that have already performed a set up procedure to securely create
```

Comment:
Is a reference to https://bitcoinops.org/en/newsletters/2024/07/19/#distributed-key-generation-protocol-for-frost applicable here?